docker-mixin/dashboards/docker-logs.libsonnet

```diff
@@ -0,0 +1,280 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local custom_barchart_grafonnet = import '../lib/custom-barchart-grafonnet/custom-barchart.libsonnet';
+
+local host_matcher = 'job=~"$job"';
+local container_matcher = host_matcher + ', container=~"$container"';
+
+local queries = {
+  total_log_lines: 'sum(count_over_time({' + container_matcher + '}[$__interval]))',
+  total_log_warnings: 'sum(count_over_time({' + container_matcher + '} |= "Warning" [$__interval]))',
+  total_log_errors: 'sum(count_over_time({' + container_matcher + '} |= "Error" [$__interval]))',
+  error_percentage: 'sum( count_over_time({' + container_matcher + '} |= "Error" [$__interval]) ) / sum( count_over_time({' + container_matcher + '} [$__interval]) )',
+  total_bytes: 'sum(bytes_over_time({' + container_matcher + '} [$__interval]))',
+  error_log_lines: '{' + container_matcher + '} |= "Error"',
+  warning_log_lines: '{' + container_matcher + '} |= "Warning"',
+  log_full_lines: '{' + container_matcher + '}',
+};
+
+local stackstyle = {
+  line: 1,
+  fill: 5,
+  fillGradient: 10,
+};
+
+// Templates
+local ds_template = {
+  current: {
+    text: 'default',
+    value: 'default',
+  },
+  hide: 0,
+  label: 'Data Source',
+  name: 'loki_datasource',
+  options: [],
+  query: 'loki',
+  refresh: 1,
+  regex: '',
+  type: 'datasource',
+};
+
+local job_template = grafana.template.new(
+  'job',
+  '$loki_datasource',
+  'label_values(job)',
+  label='Job',
+  refresh='load',
+  multi=true,
+  includeAll=true,
+  allValues='.+',
+  sort=1,
+  regex='.*docker.+'
```

Comment:
Is this necessary? Perhaps too opinionated, since users should be free to name their jobs whatever they like, and still find them here.

Reply:
Yup removed the regex based on our discussion